### DISCUSSION TITLE
Add entity state to lawn_mower for returning to dock

### DISCUSSION THREAD

**Discussion by andyb2000** [home-assistant/architecture#1122] [Entity Models]
I've opened a PR https://github.com/home-assistant/core/pull/123368
this is to add the option for RETURNING to the lawn_mower entity. This allows it to return state of returning to dock similar to vacuum like entities.

At present this state does not exists so the user is unaware of the mower in a return to dock status.

**Comment by joostlek** [comment]
Please add a list of integrations that can use this feature

**Comment by andyb2000** [comment]
ignore this.
